source/shared/sheets.js

```diff
@@ -32,19 +32,34 @@ const ARCHIVE_CONTENTS_ADD_ITEM_SHEET_BUTTONS = [
 ];
 const ARCHIVES_PAGE_RIGHT_SHEET_BUTTONS = [SHEET_ADD_ARCHIVE, SHEET_LOCK_ALL, SHEET_CANCEL];
 
+function removeTextFromArray(arr, text) {
```

Comment:
Can this use `filter` perhaps? (optional :D)
```js
function removeTextFromArray(arr, text) {
	return arr.filter(item => item !== text);
}
```

Reply:
Filter isn't destructive, and I was working on a const in this case. Filter might also be a bit more verbose as I'm calling it repeatedly. There might be a nicer way though, let me think on it..